### DISCUSSION TITLE
PI3 + LCD + DAC

### DISCUSSION THREAD

**Discussion by Piattica** [probonopd/MiniDexed#552] [Q&A]
Ciao from Italy
I am a newbie about Raspberry. I have a PI3 and I bought a PiFi DAC+ v2.0 that has to be connected with 20x2 connector on PI3.
Question: how can I connect LCD and button now? Is it safe to add a [GPIO duplicator](https://www.reichelt.com/it/it/raspberry-pi-testata-gpio-1-su-2-40-pin-rm-2-54-rpi-gpio-1to2-p276993.html?PROVID=2814&gclid=Cj0KCQjwy4KqBhD0ARIsAEbCt6gDhYgXa6Z4Xs1RTAqLiDg4NFmsj59zPWFobmq1oTToDoue1iRHMEAaAtUsEALw_wcB) between PI3 and PiFi DAC+ v2.0 or what else?

**Comment by Banana71** [comment]
Basically it's as possible as you want it to be. Depending on which PINs your DAC uses, you may have to adjust the GPIO for the encoder and/or the display in the minidexed.ini.
I don't want to talk you out of your plan, but you should note that the GPIO adapter costs more than twice as much as a DAC GY-5102 on Aliexpress. With the adapter, the spacers between your Raspberry Pi and the DAC will no longer fit.
